lib/common/version.go

```diff
@@ -0,0 +1,6 @@
+package common
+
+const (
+	TransactionMessage_Version1 = "tx_version1"
+	BallotMessage_Version1      = "ballot_version1"
+)
```

Comment:
@soonkuk We talked about version, the version value should be `1`, not `tx_version1`